engine/gamesys/src/gamesys/components/comp_collision_object.cpp

```diff
@@ -956,50 +1011,40 @@ namespace dmGameSystem
         step_world_context.m_RayCastCallback = RayCastCallback;
         step_world_context.m_RayCastUserData = world;
 
-        world->m_LastDT = params.m_UpdateContext->m_DT;
+        float physics_dt = params.m_UpdateContext->m_DT;
+        uint32_t num_steps_i = 1;
 
-        g_NumPhysicsTransformsUpdated = 0;
+        static int frame = 0;
```

Comment:
What is this used for?

Reply:
It's a remnant of a debug print. will remove